playbooks/roles/caasp4/templates/skuba-terraform.tfvars.j2

```diff
@@ -0,0 +1,60 @@
+# This file is managed by ansible and based on terraform.tfvars.ci.example from github.com/SUSE/skuba
+image_name = "{{ image_name }}"
+internal_net = "{{ internal_net }}"
+internal_subnet = "{{ internal_subnet }}"
+internal_router = "{{ internal_router }}"
+external_net = "{{ external_net }}"
+stack_name = "{{ stack_name }}"
+subnet_cidr = "{{ subnet_cidr }}"
+"dns_nameservers" = [
+    "8.8.8.8",
+    "8.8.8.4"
+]
+masters = 1
+workers = 2
+master_size = "m1.medium"
+worker_size = "m1.medium"
+
+username = "sles"
+password = "linux"
+
+repositories = [
+  {
+    caasp_40_devel_sle15sp1 = "http://download.suse.de/ibs/Devel:/CaaSP:/4.0/SLE_15_SP1/"
+  },
+  {
+    sle15sp1_pool = "http://download.suse.de/ibs/SUSE:/SLE-15-SP1:/GA/standard/"
+  },
+  {
+    sle15sp1_update = "http://download.suse.de/ibs/SUSE:/SLE-15-SP1:/Update/standard/"
+  },
+  {
+    sle15_pool = "http://download.suse.de/ibs/SUSE:/SLE-15:/GA/standard/"
+  },
+  {
+    sle15_update = "http://download.suse.de/ibs/SUSE:/SLE-15:/Update/standard/"
+  },
+  {
+    suse_ca = "http://download.suse.de/ibs/SUSE:/CA/SLE_15_SP1/"
+  }
+]
+
+packages = [
+  "kernel-default",
+  "-kernel-default-base",
+  "ca-certificates-suse",
+  "patterns-caasp-Node",
+]
+
+authorized_keys = [
+{% for key in authorized_keys %}
+  "{{ key }}",
+{% endfor %}
+]
+
+ntp_servers = [
+{% for server in ntp_servers %}
```

Comment:
same as authorized keys

Reply:
I think it's not the same. I want a new line per entry

Reply:
Got it. It's cleaner this way then.